docs/plugins/spotify-connect.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Spotify Connect Plugin
+description: Features and Notes for the Spotify Connect Plugin
+---
+
+# Spotify Connect ![Preview image](../assets/icons/spotify-connect-icon.png){ align=right }
+
+Music Assistant has the ability to add [Spotify Connect](https://connect.spotify.com/) support to any MA player.
+
+## Features
+
+- Any MA player can be exposed including groups
+
+## Known Issues / Notes
+
```

Comment:
- Because of the universal, buffered nature of forwarding the Spotify Connect audio stream to MA players, there will be a delay (of 2 up to 20 seconds) in sending a command from the spotify app and metadata can also be ahead of time.

Reply:
done